sdk/textanalytics/Azure.AI.TextAnalytics/src/LinkedEntity.cs

```diff
@@ -62,7 +62,7 @@ internal LinkedEntity(string name, IEnumerable<LinkedEntityMatch> matches, strin
 
         /// <summary> Bing Entity Search API unique identifier of the recognized entity.
         /// Use in conjunction with the Bing Entity Search API to fetch additional relevant information.
-        /// This property is only available for v3.1-preview.2 and up.
+        /// This property is only available for service v3.1-preview.2 and up.
```

Comment:
nit: service API version v3.1-preview.2 and up.

I've also noticed you refer to the latest preview version as both the generic v3.1-preview, and v3.1-preview.2. Don't have a strong preference over which one you use, but think they should be consistent. Python uses v3.1-preview to refer to latest service version

Reply:
Fair. modified to `v3.1-preview and up.`